runit.fish

```diff
@@ -0,0 +1,2 @@
+node run.js ../heroku-apps ../heroku-certs ../heroku-ci ../heroku-cli-addons ../heroku-cli-oauth ../heroku-container-registry ../heroku-fork ../heroku-git ../heroku-local ../heroku-orgs ../heroku-pg ../heroku-pipelines ../heroku-ps-exec ../heroku-run ../heroku-spaces ../heroku-status
```

Comment:
we could add them as devDependencies as well as peerDependencies so when we use it as a script we don't have to have exact versions